src/module_1/module_1_meteo_api.py

```diff
@@ -1,7 +1,83 @@
-""" This is a dummy example """
+import requests
+import time
+from datetime import datetime, timedelta
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
 
+
+API_URL = "https://climate-api.open-meteo.com/v1/climate?"
+VARIABLES = "temperature_2m_mean,precipitation_sum,soil_moisture_0_to_10cm_mean"
+COORDINATES = {
+    "Madrid": {"latitude": 40.416775, "longitude": -3.703790},
+    "London": {"latitude": 51.507351, "longitude": -0.127758},
+    "Rio": {"latitude": -22.906847, "longitude": -43.172896},
+}
+# Funcion para descargar los datos de la API
```

Comment:
follow the guidelines to comment functions in python: https://stackoverflow.com/questions/2357230/what-is-the-proper-way-to-comment-functions-in-python